README.md

```diff
@@ -148,3 +148,23 @@ of node sets which are specified by the user.
     apply_set_order:
       - canaries
       - all
+
+### Building/Running Hecklerd on Mac
```

Comment:
Is there a reason to document this and not update the Makefile to build on macos?

```
UNAME := $(shell uname)

ifeq ($(UNAME), Linux)
# do something Linux-y
endif
ifeq ($(UNAME), Darwin)
# do something Macos-y
endif
```

Reply:
Unable to compile and install musl-gcc package/binary for the latest mac OS. 

With the above instructions, developers can build libigt2 in dynamic share mode and run hecklerd for their testing and validation purposes. 
For Prod/QA deployment builds, Makefile can be used to build binary files.